scripts/checks.sh

```diff
@@ -563,5 +589,25 @@ function run_checks()
 	| jq -s 'add | {checks:.}'
 }
 
+# Parse arguments
+while [[ $# -gt 0 ]]; do
+    arg="$1"
+
+    case $arg in
+	-h|--help)
+	    help
+	    exit 0
+	    ;;
+        --resinos-registry | --balenaos-registry)
```

Comment:
let's drop `--resinos` support unless there's a good reason to keep it around (i don't believe there is)